### DISCUSSION TITLE
What does this do?

### DISCUSSION THREAD

**Discussion by ShadowbreakerGD** [FlagClicked/FlagClicked#39] [Help]
I’m new to GitHub, and I’m still learning what I need to learn. [I was able to use replit and host the repo](https://flagclicked.shadowbreakergd.repl.co/) but I can get it to do anything, is there something else I need to do? Like edit some of the code? I do use scratch to code sometimes, but, like, what is this for? 

Thanks, Shadow ![A7443A3A-BB47-498F-B099-3F839C63E4E4](https://user-images.githubusercontent.com/78216950/113365989-e9c4e480-9325-11eb-83aa-739e84157cab.png)

**Comment by ShadowbreakerGD** [comment]
Oh, hold on I think I’ve got it...

**Comment by ShadowbreakerGD** [comment]
Nevermind! I’ve figured it out! XD

Thanks anyway, Shadow ![5825BD14-08B1-427D-B7EE-6777E2968D55](https://user-images.githubusercontent.com/78216950/113366526-73c17d00-9327-11eb-879f-6017fdd9bc78.png)

**Reply by danielgallab** [reply]
Glad you got it xD
